### DISCUSSION TITLE
Write more than one result file

### DISCUSSION THREAD

**Discussion by tormento** [Breakthrough/PySceneDetect#443] [Q&A] [answered]
QP file is a blessing.

Sometimes I need both QP and scene changes times.

Not to compute the video twice, would you please add multiple result file output?

**Comment by Breakthrough** [comment, marked as answer]
PySceneDetect supports outputting as many files types as you wish. You can use both `export-qp` and `list-scenes` together.

**Comment by tormento** [comment]
Ah, ok. Didn't realize that.
